src/core/client/admin/routes/ControlPanel/FlushRedisMutation.ts

```diff
@@ -0,0 +1,34 @@
+import { graphql } from "react-relay";
+import { Environment } from "relay-runtime";
+
+import {
+  commitMutationPromiseNormalized,
+  createMutation,
+  MutationInput,
+} from "coral-framework/lib/relay";
+
+import { FlushRedisMutation as MutationTypes } from "coral-admin/__generated__/FlushRedisMutation.graphql";
+
+let clientMutationId = 0;
+
+const FlushRedisMutation = createMutation(
+  "invalidateCachedStory",
```

Comment:
Can you update to match that this is the flush Redis mutation?

Reply:
Oops! Yup!